indra/newview/app_settings/settings.xml

```diff
@@ -3739,6 +3739,28 @@
       <key>Value</key>
       <integer>1</integer>
     </map>
+    <key>EnableGameControlInput</key>
+    <map>
+      <key>Comment</key>
+      <string>Transmit game controller input to server</string>
+      <key>Persist</key>
+      <integer>1</integer>
+      <key>Type</key>
+      <string>Boolean</string>
+      <key>Value</key>
+      <integer>0</integer>
+    </map>
+    <key>EnableGameControlKeyboardInput</key>
+    <map>
+      <key>Comment</key>
+      <string>Send 'unhandled' keystrokes as GameInput to server</string>
+      <key>Persist</key>
+      <integer>1</integer>
+      <key>Type</key>
+      <string>Boolean</string>
+      <key>Value</key>
+      <integer>0</integer>
+    </map>
```

Comment:
The initial UI for controlling this feature is very rudimentary to start.  Two checkboxes: one to enable the stream of data, and another to consider the keyboard to be a game controller with lots of buttons.